README.md

```diff
@@ -161,6 +161,11 @@ You can change yb to other channels by using `-channel`:
 
 Currently, the _inner_ yb that runs inside the build container will use the stable channel by default. To change that, set the environment variable `YB_UPDATE_CHANNEL=unstable` before running yb.
 
+# Mac OS X dependency
```

Comment:
Let's add this to the "How to use it" section so the user isn't surprised later.